Packs/CortexXDR/Scripts/InvestigationVerdict/README.md

```diff
@@ -0,0 +1,21 @@
+This script displays the Verdict.
```

Comment:
the verdict of what? based of which field?

Reply:
Done